### ISSUE TITLE
[a11y] 11.1.3.1a Info and relationships (headings)

### ISSUE THREAD

**Issue by tschiebel** [owncloud/android#4362]
audit: https://infinite.owncloud.com/s/QtwuiGHikwFjozV

https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf
https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de

### Shares

- [x] There are several visually clearly recognizable headings on the page, but they are not
labeled as such. These headings include "Sharing", "Users
and groups" and "Public links".

### Edit shared link

- [x] There is a clearly visually recognizable heading on the page "Edit shared link
Edit shared link" on the page, but it has not been labeled as such.

### Spaces

- [x] There is a clearly visually recognizable heading "Spaces" on the page, but it has not been
not labeled as such.

Note: The heading "Spaces" also functions as a button, but this is not visually recognizable.
visually recognizable. It would be better to provide a separate button for editing
button.

### Manage accounts

- [x] There is a clearly visually recognizable heading "Manage accounts" on the page, which
not labeled as such.

### Image Viewer

- [x] The file name could be labeled as a heading.

**Comment by jesmrec** [comment]
Need research: how to set text as heading in Android. Don't seem to be something native in the platform. 

As posible plan B: add "heading" prefix in alternative text.

- Translatable strings

**Comment by JuancaG05** [comment]
Check together with https://github.com/owncloud/android/issues/4363, https://github.com/owncloud/android/issues/4371 and https://github.com/owncloud/android/issues/4373

**Comment by Aitorbp** [comment]
> audit: https://infinite.owncloud.com/s/QtwuiGHikwFjozV
> 
> https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
> 
> ### Shares
> * [ ]  There are several visually clearly recognizable headings on the page, but they are not
>   labeled as such. These headings include "Sharing", "Users
>   and groups" and "Public links".

The heading has been included in the element of the Shares: "toolbar" title, "Users and groups" and "Public links".
 
> ### Edit shared link
> * [ ]  There is a clearly visually recognizable heading on the page "Edit shared link
>   Edit shared link" on the page, but it has not been labeled as such.

The heading has been added to Edit shared link title on the top of the dialog. 

> ### Spaces
> * [ ]  There is a clearly visually recognizable heading "Spaces" on the page, but it has not been
>   not labeled as such.

> Note: The heading "Spaces" also functions as a button, but this is not visually recognizable. visually recognizable. It would be better to provide a separate button for editing button.
> 

In this case, it is not going to be labeled as a heading, because we consider it to be a specific element within the toolbar and there are a lot of elements in that view. Also, its functionality is not that of a header, but that of a search item. The Search view could be changed to only open by clicking the search view icon, but that is outside the scope of the issue. Therefore, it will not be changed either.

> ### Manage accounts
> * [ ]  There is a clearly visually recognizable heading "Manage accounts" on the page, which
>   not labeled as such.
Manage accounts title has been added as a heading.

> ### Image Viewer
> * [ ]  The file name could be labeled as a heading.
As for the file name of the image viewer, it is not going to be labeled as a heading, because we think it to be a specific item within the toolbar and there are more elements in that view (five exactly). Same logic like the Spaces.
